### PR TITLE
🌱 Fix spelling in log message (cluter -> cluster)

### PR DESCRIPTION
**Pull request by oscr**
**What this PR does / why we need it**:

Found a spelling mistake in the logs and making a separate pr so it can be cherry-picked into older releases.

### CODE REVIEW

File: cmd/clusterctl/client/cluster/mover.go [hunk 1]
```diff
@@ -443,7 +443,7 @@ func (o *objectMover) restore(graph *objectGraph, toProxy Proxy) error {
 
 	// Resume reconciling the ClusterClasses after being restored from a backup.
 	// By default, during backup, ClusterClasses are paused so they must be unpaused to be used again
-	log.V(1).Info("Resuming the target cluter classes")
+	log.V(1).Info("Resuming the target cluster classes")
```
Comment (killianmuldoon) [inline cmd/clusterctl/client/cluster/mover.go:446]: nit
```suggestion
	log.V(1).Info("Resuming the target ClusterClasses")
```

And the same elsewhere 🙂

**Review by killianmuldoon — COMMENTED** [review]
Looks good with one nit.

**Review by fabriziopandini — COMMENTED** [review]
/lgtm

**Review by killianmuldoon — COMMENTED** [review]
/lgtm

**Review by ykakarap — APPROVED** [review]
/lgtm

**Review by fabriziopandini — COMMENTED** [review]
/approve

**Pull request merged**